packages/CopyInput/src/CopyInput.js

```diff
@@ -100,18 +112,25 @@ const propTypes = {
   isReadOnly: PropTypes.bool,
   /** Default value for the input */
   value: PropTypes.string,
+  /** Should hide the value text */
```

Comment:
Maybe something a bit more explanatory, since this isn't so obvious.  Like, 
> If a plain text version of the value will be rendered

Then maybe for `hasInputContainer`:
> If the value will be rendered in an Input component